sql/admin.md

```diff
@@ -143,6 +143,44 @@ ADMIN CANCEL DDL JOBS job_id [, job_id] ...
 
 用于查看当前 DDL 作业队列中的所有结果（包括正在运行以及等待运行的任务）以及已执行完成的 DDL 作业队列中的最近十条结果。
 
+  ```sql
```

Comment:
- Delete the extra spaces at the beginning of the first 3 lines.
- Indent the whole code block with 4 spaces, because it is an example of `ADMIN SHOW DDL JOBS`.